lib/rules/validate-jsdoc/require-description-complete-sentence.js

```diff
@@ -0,0 +1,62 @@
+module.exports = requireDescriptionCompleteSentence;
+module.exports.scopes = ['function'];
+module.exports.options = {
+    requireDescriptionCompleteSentence: {allowedValues: [true]}
+};
+
+var RE_START_WITH_UPPER_CASE = /^[A-Z]/;
+
+/**
+ * Checks that a period is next to a word at the end of input.
+ *
+ * The sentence ending with a new line is also valid.
+ */
+var RE_END_WITH_PERIOD = /\w[.](\n|$)/;
+
+var RE_END_DESCRIPTION = /\n/g;
+
+var RE_LAST_WORD = /\w(?!.*\w)/;
+
+/**
+ * Requires description to be a complete sentence in a jsdoc comment.
+ *
+ * A complete sentence is defined by starting with an upper letter
+ * and ending with a period.
+ *
+ * @param {(FunctionDeclaration|FunctionExpression)} node
+ * @param {Function} err
+ */
+function requireDescriptionCompleteSentence(node, err) {
+    var doc = node.jsdoc;
+    if (!doc || !doc.tags.length || !doc.description || !doc.description.length) {
+        return;
+    }
+
+    var loc = doc.loc.start;
+
+    if (!RE_START_WITH_UPPER_CASE.test(doc.description)) {
+        err('Sentence must start with an upper case letter.', {
+            line: loc.line + 1,
+            column: loc.column + 3
+        });
+    }
+
+    if (!RE_END_WITH_PERIOD.test(doc.description)) {
```

Comment:
I was actually going to do that but then I decided against it because I am adding a new test case for next line starting with an upper case letter.

If more cases are added someone might put another case after the return and then it may not run if it has nothing to do with the documentation ending with a period.